docs/watch-options.md

```diff
@@ -203,7 +203,7 @@ Set `options: { cwd: { files: 'a/path', event: 'a/path' }}` to strip off `a/path
 
 
 ## options.livereloadOnError
-Type: `Boolean`  
-Default: `true`  
+Type: `Boolean`
```

Comment:
And in fact it's not missing in line 197; I fixed those the other day https://github.com/gruntjs/grunt-contrib-watch/commit/2345454e44c42e1639a2b79a006909ff08feefcb